SERVER/static/js/chat.js

```diff
@@ -21,20 +21,36 @@
   };
 
   sendMessage = function() {
```

Comment:
Send message should take a packet and send it... not create a packet and send it.